.npmrc

```diff
@@ -0,0 +1 @@
+//webclients1-nexus.service.cd-dev.consul/repository/npm-internal/:_authToken=6fc043ce-9d5d-3c02-aa3b-234de4554823
```

Comment:
checking in authorisation tokens into GH?

Reply:
I didn't realize that this file was checked in.